app/models/item.rb

```diff
@@ -5,6 +5,4 @@ class Item < ApplicationRecord
 
   validates_presence_of :name, :description, :unit_price
   validates_numericality_of :unit_price, only_integer: true
-
-  enum status: [:disabled, :enabled]
```

Comment:
Why was the item enabled/disabled removed? I thought it was necessary for user story 9

Reply:
good eye, it should be included